views/partials/header.ejs

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>TITLE</title>
+  <title>Who represents me?</title>
```

Comment:
This change fell into this branch. If we had, for example, debated the title in the other PR and changed it there, you'd be overwriting it when we merge this PR. No action for this right now since it now effectively matches master.